src/pdm/cli/commands/new.py

```diff
@@ -11,8 +11,8 @@ class Command(InitCommand):
 
     arguments = (verbose_option,)
 
-    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
-        super().add_arguments(parser)
+    def add_arguments(self, parser: argparse.ArgumentParser, is_init=False) -> None:
```

Comment:
i would prefer not to modify the method signature, but use a class attribute `supports_other_generator`

and you will also need to remove --copier and --cookiecutter for pdm new

Reply:
ok!